conf/dist/env.ac

```diff
@@ -19,11 +19,11 @@ AC_CCACHE=true
 
 OUTPUT_FOLDER=/azerothcore/var/build/sql/
 
-DB_AUTH_CONF=MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';
+DB_AUTH_CONF=MYSQL_USER="'root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';"
```

Comment:
" on wrong plaace, should be "MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';"

Reply:
fixed!